src/main/java/uk/gov/hmcts/ccd/endpoint/exception/RestExceptionHandler.java

```diff
@@ -40,4 +42,13 @@ public RestExceptionHandler(AppInsights appInsights) {
         LOG.warn("exception with message {}, {}, {}", errorMsg, HttpStatus.INTERNAL_SERVER_ERROR, request, e);
         return handleExceptionInternal(e, errorMsg, new HttpHeaders(), HttpStatus.INTERNAL_SERVER_ERROR, request);
     }
+
+    @ExceptionHandler(SQLException.class)
+    protected ResponseEntity<Object> handleSQLException(final SQLException e, final WebRequest request) {
+        final String errorMsg = "SQL Exception thrown during API operation, " + HttpStatus.INTERNAL_SERVER_ERROR;
```

Comment:
Appending HttpStatus.INTERNAL_SERVER_ERROR at end of error message didn't look useful. Error message can just tell what. The fact that it's returned as a http-500 doesn't need to be mentioned.